### PR TITLE
DF-1915 removing sbn35 code due to chisco migration

### PR DESCRIPTION
**Pull request by hamidsoleymani**
removed sbn35 code

### CODE REVIEW

File: domain/src/main/java/ch/admin/seco/jobs/services/jobadservice/domain/jobadvertisement/Occupation.java [hunk 1]
```diff
@@ -107,8 +91,6 @@ public boolean equals(Object o) {
         if (o == null || getClass() != o.getClass()) { return false; }
         Occupation that = (Occupation) o;
         return Objects.equals(avamOccupationCode, that.avamOccupationCode) &&
-                Objects.equals(sbn3Code, that.sbn3Code) &&
-                Objects.equals(sbn5Code, that.sbn5Code) &&
                 Objects.equals(bfsCode, that.bfsCode) &&
```
Comment (cjpsycos) [inline domain/src/main/java/ch/admin/seco/jobs/services/jobadservice/domain/jobadvertisement/Occupation.java]: shouldn't chisco-code be compared?

File: domain/src/main/java/ch/admin/seco/jobs/services/jobadservice/domain/jobadvertisement/Occupation.java [hunk 2]
```diff
@@ -118,15 +100,15 @@ public boolean equals(Object o) {
 
     @Override
     public int hashCode() {
-        return Objects.hash(avamOccupationCode, sbn3Code, sbn5Code, bfsCode, label, workExperience, educationCode, qualificationCode);
+        return Objects.hash(avamOccupationCode, bfsCode, label, workExperience, educationCode, qualificationCode);
```
Comment (cjpsycos) [inline domain/src/main/java/ch/admin/seco/jobs/services/jobadservice/domain/jobadvertisement/Occupation.java]: shouldn't chisco-code be hashed?

File: domain/src/test/java/ch/admin/seco/jobs/services/jobadservice/domain/jobadvertisement/fixture/OccupationFixture.java [hunk 3]
```diff
@@ -8,8 +8,6 @@
     public static Builder testOccupation() {
         return testOccupationEmpty()
         .setAvamOccupationCode("9999")
-        .setSbn3Code("sbn3Code")
```
Comment (cjpsycos) [inline domain/src/test/java/ch/admin/seco/jobs/services/jobadservice/domain/jobadvertisement/fixture/OccupationFixture.java]: shouldn't chisco-code be set?

Comment (hamidsoleymani) [reply domain/src/test/java/ch/admin/seco/jobs/services/jobadservice/domain/jobadvertisement/fixture/OccupationFixture.java]: chisco for this test is not needed

File: domain/src/main/java/ch/admin/seco/jobs/services/jobadservice/domain/jobadvertisement/Occupation.java (hunk 1 above)
Comment (hamidsoleymani) [reply domain/src/main/java/ch/admin/seco/jobs/services/jobadservice/domain/jobadvertisement/Occupation.java]: I will add chisco for equals

File: domain/src/main/java/ch/admin/seco/jobs/services/jobadservice/domain/jobadvertisement/Occupation.java (hunk 2 above)
Comment (hamidsoleymani) [reply domain/src/main/java/ch/admin/seco/jobs/services/jobadservice/domain/jobadvertisement/Occupation.java]: I will add chisco for equals

**Pull request merged**